### PR TITLE
[MOS-325] AudioServiceAPI removed from Call app

### PR DESCRIPTION
**Pull request by kkleczkowski**
AudioServiceAPI is now removed from Call app.
Call audio routing is handled in service cellular.

**Description**

Please describe your pull request

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [x] Has documentation updated

Thanks for your work ♥

### CODE REVIEW

File: module-services/service-cellular/call/CellularCall.cpp [hunk 1]
```diff
@@ -186,6 +186,24 @@ namespace CellularCall
...
+            break;
+        case cellular::CallAudioEventRequest::EventType::Unmute:
+            audio.unmouteCall();
+            break;
+        case cellular::CallAudioEventRequest::EventType::LoudspeakerOn:
+            audio.setLaudSpeakerOn();
+            break;
+        case cellular::CallAudioEventRequest::EventType::LoudSpeakerOff:
+            audio.setLaudSpeakerOff();
+            break;
```
Comment (SP2FET) [inline module-services/service-cellular/call/CellularCall.cpp:203]:
```suggestion
            audio.setLoudspeakerOn();
            break;
        case cellular::CallAudioEventRequest::EventType::LoudspeakerOff:
            audio.setLoudspeakerOff();
            break;
```
Loudspeaker is one word, so it shouldn't be capitalized this way: LoudSpeaker

File: module-services/service-cellular/service-cellular/CellularMessage.hpp [hunk 2]
```diff
@@ -1028,4 +1028,20 @@ namespace cellular
...
+
+    class CallAudioEventRequest : public sys::DataMessage
+    {
+      public:
+        enum class EventType
+        {
+            Mute,
+            Unmute,
+            LoudspeakerOn,
+            LoudSpeakerOff
```
Comment (SP2FET) [inline module-services/service-cellular/service-cellular/CellularMessage.hpp:1040]:
```suggestion
            LoudspeakerOn,
            LoudspeakerOff
```

File: module-services/service-cellular/call/CellularCall.cpp [hunk 3]
```diff
@@ -186,6 +186,24 @@ namespace CellularCall
...
     }
 
+    void Call::handleCallAudioEventRequest(cellular::CallAudioEventRequest::EventType event)
+    {
+        switch (event) {
+        case cellular::CallAudioEventRequest::EventType::Mute:
+            audio.muteCall();
+            break;
+        case cellular::CallAudioEventRequest::EventType::Unmute:
+            audio.unmouteCall();
```
Comment (SP2FET) [inline module-services/service-cellular/call/CellularCall.cpp:196]:
```suggestion
            audio.unmuteCall();
```

**Review by pholat — APPROVED** [review]

**Review by Maciej-Mudita — APPROVED** [review]

File: module-services/service-cellular/call/CellularCall.cpp (hunk 3 above)
Comment (kkleczkowski) [reply module-services/service-cellular/call/CellularCall.cpp]: done

File: module-services/service-cellular/service-cellular/CellularMessage.hpp (hunk 2 above)
Comment (kkleczkowski) [reply module-services/service-cellular/service-cellular/CellularMessage.hpp]: done

**Pull request merged**